trl/trainer/sft_trainer.py

```diff
@@ -24,6 +24,7 @@
 from datasets import Dataset
 from datasets.arrow_writer import SchemaInferenceError
 from datasets.builder import DatasetGenerationError
+from huggingface_hub.utils._deprecation import _deprecate_arguments
```

Comment:
Is it safe to use a private method ? Are we expecting any potential breaking change here?